lib/client_test.go

```diff
@@ -335,6 +335,88 @@ func TestClient_Query(t *testing.T) {
 	}
 }
 
+func TestClient_Add(t *testing.T) {
```

Comment:
Please use `Helper` for teardown to delete all data.
You can use `DeleteAll`.
It may have the effect to other test cases if you have made some changes in a test case.

Reply:
I had to create another `Helper` object to get this to work (the global one throws a `the client connection is closing` error). Is this okay?

Reply:
Yes, please replace the helper to use it!

Reply:
Ooh got it now. Did that.